### PR TITLE
introduce queued_at field in payload

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**

### CODE REVIEW

File: lib/travis/scheduler/serialize/worker.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def data
             repository: repository_data,
             ssh_key: ssh_key,
             timeouts: repo.timeouts,
-            cache_settings: cache_settings
+            cache_settings: cache_settings,
```
Comment (meatballhat) [inline lib/travis/scheduler/serialize/worker.rb]: I'm pretty sure rubocop disagrees with this move, but I see that this project doesn't have rubocop anymore, so .. 😐

File: spec/travis/scheduler/serialize/worker_spec.rb [hunk 2]
```diff
@@ -86,7 +87,7 @@ def encrypted(value)
           hard_limit: 180 * 60, # worker handles timeouts in seconds
           log_silence: 20 * 60
         },
-        cache_settings: s3
+        cache_settings: s3,
```
Comment (meatballhat) [inline spec/travis/scheduler/serialize/worker_spec.rb]: Same here.  Adding trailing commas for funsies? 😃

File: spec/travis/scheduler/serialize/worker_spec.rb [hunk 3]
```diff
@@ -147,8 +148,9 @@ def encrypted(value)
           state: 'queued',
           secure_env_enabled: false,
           debug_options: {},
+          queued_at: '2016-01-01T10:30:00Z',
           pull_request_head_branch: 'head_branch',
-          pull_request_head_sha: '12345'
+          pull_request_head_sha: '12345',
```
Comment (meatballhat) [inline spec/travis/scheduler/serialize/worker_spec.rb]: 👀

File: spec/travis/scheduler/serialize/worker_spec.rb [hunk 4]
```diff
@@ -175,7 +177,7 @@ def encrypted(value)
           hard_limit: 180 * 60, # worker handles timeouts in seconds
           log_silence: 20 * 60
         },
-        cache_settings: s3
+        cache_settings: s3,
```
Comment (meatballhat) [inline spec/travis/scheduler/serialize/worker_spec.rb]: 👀

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
The trailing commas were added while introducing keys and then moving them around. The use of trailing commas is not regulated and already inconsistent in this file. So unless there is a major reason to make that all consistent now, I'll leave it as is if that's ok. :)

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
After looking at the initial patch it had a bias towards no trailing commas, so I'm changing my mind and changing that. :)

**Pull request merged**